tools/kolla-ansible

```diff
@@ -123,28 +123,29 @@ Options:
     --verbose, -v                      Increase verbosity of ansible-playbook
 
 Commands:
-    prechecks           Do pre-deployment checks for hosts
-    check               Do post-deployment smoke tests
-    mariadb_recovery    Recover a completely stopped mariadb cluster
-    mariadb_backup      Take a backup of MariaDB databases
-                            --full (default)
-                            --incremental
-    bootstrap-servers   Bootstrap servers with kolla deploy dependencies
-    destroy             Destroy Kolla containers, volumes and host configuration
-                            --include-images to also destroy Kolla images
-                            --include-dev to also destroy dev mode repos
-    deploy              Deploy and start all kolla containers
-    deploy-bifrost      Deploy and start bifrost container
-    deploy-servers      Enroll and deploy servers with bifrost
-    deploy-containers   Only deploy and start containers (no config updates or bootstrapping)
-    post-deploy         Do post deploy on deploy node
-    pull                Pull all images for containers (only pulls, no running container changes)
-    reconfigure         Reconfigure OpenStack service
-    stop                Stop Kolla containers
-    certificates        Generate self-signed certificate for TLS *For Development Only*
-    upgrade             Upgrades existing OpenStack Environment
-    upgrade-bifrost     Upgrades an existing bifrost container
-    genconfig           Generate configuration files for enabled OpenStack services
+    prechecks            Do pre-deployment checks for hosts
```

Comment:
What caused this big diff?

Reply:
whitespace, maybe?

Reply:
`ocavia-certificates` was added as a command, and it became the longest command in the first column, so all the descriptions had to shift, I think.